content/en/docs/ops/ambient/getting-started/index.md

```diff
@@ -214,6 +214,11 @@ $ sed -e 's/from: Same/from: All/'\
 ' @samples/bookinfo/gateway-api/bookinfo-gateway.yaml@ | kubectl apply -f -
 {{< /text >}}
 
+{{< warning >}}
+Kindly make sure you have a IP address for the bookinfo-gateway service in Kind, you can install [MetalLB](https://kind.sigs.k8s.io/docs/user/loadbalancer/)
```

Comment:
```suggestion
To get IP address assignments for `LoadBalancer` service types in kind, you may need to install a tool like [MetalLB](https://metallb.universe.tf/). Please consult [this guide](https://kind.sigs.k8s.io/docs/user/loadbalancer/) for more information.
```

Reply:
Linking to a Kind doc with text of `MetalLB` is kind of confusing, links with text of MetalLB should be to MetalLB.

Reply:
Ah yes, that's there, will update